src/layout/dom.ts

```diff
@@ -156,6 +185,34 @@ export function update(node: Node, data: NodeInfo, source: Source): void {
     }
 }
 
+function mask(data: NodeInfo, masked: boolean): boolean {
+    let attributes = data.attributes;
+    let tag = data.tag.toUpperCase();
+
+    // Do not proceed if attributes are missing for the node
+    if (attributes === null || attributes === undefined) { return masked; }
+
+    // Check for blacklist fields (e.g. address, phone, etc.) only if the input node is not already masked
+    if (masked === false && tag === Constant.TAG_INPUT && Constant.NAME_ATTRIBUTE in attributes) {
+        let value = attributes[Constant.NAME_ATTRIBUTE].toLowerCase();
+        for (let name of BLACKLIST_NAMES) {
```

Comment:
>BLACKLIST [](start = 25, length = 9)

Change the term "blacklist" to "disallowed list" or similar. Not considered a naming "best practice" anymore.

Reply:
Done, missed it before.